Mage.Client/src/main/java/org/mage/card/arcane/ModernSplitCardRenderer.java

```diff
@@ -0,0 +1,329 @@
+package org.mage.card.arcane;
+
+import mage.ObjectColor;
+import mage.abilities.costs.mana.ManaCosts;
+import mage.constants.CardType;
+import mage.view.CardView;
+
+import java.awt.*;
+import java.awt.geom.Rectangle2D;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by StravantUser on 2017-03-30.
+ */
+public class ModernSplitCardRenderer extends ModernCardRenderer {
+
+    private class HalfCardProps {
+        int x, y, w, h, cw, ch;
+
+        String name;
+        String manaCostString;
+        ObjectColor color;
+        ArrayList<TextboxRule> rules = new ArrayList<>();
+        ArrayList<TextboxRule> keywords = new ArrayList<>();
+    }
+
+    private static ArrayList<CardType> ONLY_LAND_TYPE = new ArrayList<CardType>() {{add(CardType.LAND);}};
```

Comment:
Has to be an array to conform to the interface that it's used for, and that interface has to be an array too because order matters for which side of the card frame has which color (Unfortunately it's not strictly consistent with normal color ordering).